packages/cli-test/src/cli/shell.ts

```diff
@@ -42,15 +42,17 @@ export const shell = {
       // Listen to data event that returns all the output and collect it
       // biome-ignore lint/suspicious/noExplicitAny: stdout can accept a variety of data
       childProcess.stdout.on('data', (data: any) => {
-        sh.output += this.removeANSIcolors(data.toString());
-        logger.verbose(`Output: ${this.removeANSIcolors(data.toString())}`);
+        const output = this.removeANSIcolors(data.toString());
+        sh.output += output;
+        logger.verbose(`Output: ${output.replace(/\r?\n$/, '')}`);
```

Comment:
Taking it to the next level with `\r?` 🪟

Reply:
Of course - always keeping the cartridge return in mind! But only sometimes 😉